tools/rws-files

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env tarantool
+
+local json = require('json')
+local fio = require('fio')
+local log = require('log').new('rws-files')
+local fiber = require('fiber')
+local http_client = require('http.client')
+
+local base_url = 'https://rws.tarantool.org'
+
+log.cfg({
+    modules = {['rws-files'] = 'debug'},
+})
+
+local function visit(ctx)
+    local dir = table.remove(ctx.next_dirs, 1)
+    assert(dir:startswith('/'), dir)
+    ctx.in_progress[dir] = true
+    ctx.visited_dirs[dir] = true
+
+    -- Fetch the page.
+    local url = base_url .. dir
+    log.debug('http get: %s', url)
+    local ok, res = pcall(http_client.get, url)
+    if not ok then
+        log.warn('http get error, retrying: %s', res)
+
+        -- Return to the queue to try again.
+        table.insert(ctx.next_dirs, dir)
+        ctx.in_progress[dir] = nil
+        ctx.visited_dirs[dir] = nil
+        return
+    end
+
+    -- We shouldn't download files.
+    --
+    -- Sadly, we can't check it using HTTP OPTIONS request type,
+    -- because it return text/html unconditionally.
+    local content_type = res.headers['content-type']
```

Comment:
What do you think about moving html parsing into a separate function?

Reply:
I'll take a look.

Reply:
I think that it would make the code more readable. However, it is not done at the moment, need some spare time. I would leave it as is for now.